### PR TITLE
fixed solidity interfaces

### PR DESCRIPTION
**Pull request by albertov19**
### Description

Clears up a bit the solidity interface part for the XCM docs. Will get back to the checklist after Draft review.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: builders/xcm/xc20/xc20.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ If you are interested in testing XCM features in the Moonbase Alpha TestNet, ple
 
 This guide will show you how to retrieve the available external XC-20s and calculate their precompile addresses for the Moonbase Alpha TestNet using Polkadot.js Apps.
 
+## The External XC-20 Solidity Interface {: #the-external-xc20-interface }
+
+The Solidity interface for External XC-20 tokens is a combination of the following two interfaces:
+
+ - [ERC-20 Interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol) — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-interface)
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:26]:
```suggestion
 - [ERC-20 Interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol){target=_blank} — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-interface){target=_blank}
```

File: builders/xcm/xc20/xc20.md [hunk 2]
```diff
@@ -19,6 +19,13 @@ If you are interested in testing XCM features in the Moonbase Alpha TestNet, ple
...
+ - [Permit Interface (EIP-712 compliant)](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/Permit.sol) — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-permit-interface)
```
Comment (eshaben) [inline builders/xcm/xc20/xc20.md:27]:
```suggestion
 - [Permit Interface (EIP-712 compliant)](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/Permit.sol){target=_blank} — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-permit-interface){target=_blank}
```

File: builders/xcm/xc20/mintable-xc20.md [hunk 3]
```diff
@@ -35,6 +35,14 @@ The breakdown of responsibilities for each role is as follows:
 |  Admin   |    X     |    ✓     |     X      |    ✓     |
 | Freezer  |    X     |    X     |     ✓      |    X     |
 
+## The Mintable XC-20 Solidity Interface {: #the-mintable-xc20-interface }
+
+The Solidity interface for Mintable XC-20 tokens is a combination of the following three interfaces:
+
+ - [ERC-20 Interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol) — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-interface)
+ - [Permit Interface (EIP-712 compliant)](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/Permit.sol) — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-permit-interface)
+ - [Mintable interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/LocalAsset.sol) — as described in the [following section](#additional-functions)
```
Comment (eshaben) [inline builders/xcm/xc20/mintable-xc20.md:44]:
```suggestion
 - [ERC-20 Interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/ERC20.sol){target=_blank} — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-interface){target=_blank}
 - [Permit Interface (EIP-712 compliant)](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/Permit.sol){target=_blank} — as described in the [XC-20 overview page](/builders/xcm/xc20/overview/#the-erc20-permit-interface){target=_blank}
 - [Mintable interface](https://github.com/PureStake/moonbeam/blob/master/precompiles/assets-erc20/LocalAsset.sol){target=_blank} — as described in the [following section](#additional-functions)
```

**Comment by albertov19** [comment]
@eshaben This is ready now 😃

**Pull request merged**